docs/module_guides/querying/node_postprocessors/node_postprocessors.md

```diff
@@ -254,7 +254,48 @@ postprocessor = RankGPTRerank(top_n=3, llm=OpenAI(model="gpt-3.5-turbo-16k"))
 postprocessor.postprocess_nodes(nodes)
 ```
 
-Full notebook guide is available [her for Van Gogh](/examples/node_postprocessor/rankGPT.ipynb).
+Full notebook guide is available [here](/examples/node_postprocessor/rankGPT.ipynb).
+
+## (Beta) Colbert Reranker
+
+Uses Colbert V2 model as a reranker to rerank documents according to the fine-grained similarity between query tokens and passage tokens. Returns the top N ranked nodes.
+
+```python
+from llama_index.postprocessor.colbert_rerank import ColbertRerank
+
+colbert_reranker = ColbertRerank(
+    top_n=5,
+    model="colbert-ir/colbertv2.0",
+    tokenizer="colbert-ir/colbertv2.0",
+    keep_retrieval_score=True,
+)
+
+query_engine = index.as_query_engine(
+    similarity_top_k=10,
+    node_postprocessors=[colbert_reranker],
+)
+response = query_engine.query(
+    query_str,
+)
+```
+
+Full notebook guide is available [here](/examples/node_postprocessor/ColbertRerank.ipynb).
+
+## (Beta) Jina Reranker
```

Comment:
There is a PR just below yours for adding Jina lol

Reply:
let us merge this pr as discussed offline